components/molecules/MainHeading/main-heading.styled.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+injectGlobal`@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Raleway&display=swap');
+
+`;
+
```

Comment:
With next.js it might be better to add this in the `_document.js`

Reply:
Document docs link for you :) https://nextjs.org/docs/advanced-features/custom-document